src/controllers/currentWarningsController.js

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable */
+const mongoose = require('mongoose');
+const userProfile = require('../models/userProfile');
+
+const currentWarningsController = function (currentWarnings) {
+  const checkForDuplicates = (currentWarning, warnings) => {
+    const duplicateFound = warnings.some(
+      (warning) => warning.warningTitle.toLowerCase() === currentWarning,
+    );
+
+    return duplicateFound;
+  };
+
+  const checkIfSpecialCharacter = (warning) => {
+    return !/^[a-zA-Z][a-zA-Z0-9]*(?: [a-zA-Z0-9]+)*$/.test(warning);
```

Comment:
If we're testing for this, then we could also just use it for primary key in our database.